Sage/carbon
# Demo site Pager issue

## Description

* Removes `currentPage` from the props forms for `Pager`, `Table` and `TableAjax`
* This value was causing problems with validations and it was easier to remove it as it is available in the rendered component

## QA Steps

You should just see the Current Page prop removed form demo site forms. [This](https://github.com/Sage/carbon/pull/1146) should already have been merged

## Related

https://github.com/Sage/carbon/pull/1146
